functions/api/document_generation.py

```
@@ -0,0 +1,67 @@
+"""
+HTTP endpoint for document generation.
+"""
+import asyncio
+import json
+from typing import Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)